packages/mgt-components/src/components/mgt-people/mgt-people.scss

```diff
@@ -33,6 +33,13 @@ mgt-people .people-person {
 :host .overflow,
 mgt-people .overflow {
   span {
-    vertical-align: middle;
+    position: relative;
+    color: #1a1a1a;
+    line-height: 16px;
+    width: 16.16px;
```

Comment:
I don't think we need or want a width on this element, it's a span that contains a dynamic length element, 16px works file for +5, but two-digit overflows take more width, leaving the width unspecified, which uses the default auto behavior is probably better